_pytest/python.py

```diff
@@ -1003,11 +1003,15 @@ def write_fixture(fixture_def):
             tw.line('    no docstring available', red=True)
 
     def write_item(item):
-        name2fixturedefs = item._fixtureinfo.name2fixturedefs
```

Comment:
This is just me and a question of preference, but I try to narrow down my try/except blocks as much as possible, so I would write instead:

```python
try:
    name2fixturedefs = item._fixtureinfo.name2fixturedefs
except AttributeError:
    return

if not name2fixturedefs:
    return
```

But really it is up to you. 👍

Reply:
good point. will do that.